src/libraries/sendtohelix.proj

```diff
@@ -18,6 +18,7 @@
          Package testing doesn't run on xunit. -->
     <EnableXunitReporter Condition="'$(TargetGroup)' != 'AllConfigurations'">true</EnableXunitReporter>
     
+    <BuildConfiguration Condition="'$(TargetGroup)' != 'netcoreapp'">netcoreapp5.0</BuildConfiguration>
```

Comment:
This change is not correct from my understanding. We're using the `BuildConfiguration` that is calculated in `Directory.Build.targets` which contains OSGroup, Configuration and Arch